### PR TITLE
Ensure jinja is up to date

### PR DESCRIPTION
**Pull request by rpelisse**

### CODE REVIEW

File: playbooks/prereqs.yml [hunk 1]
```diff
@@ -0,0 +1,15 @@
...
+      - python2-pip
+  tasks:
+    - name: "Ensures the 'pip' is available to Python"
+      include_role:
+        name: 3trains_fastpackages
+  
+    - name: "Ensures jinja2 is up to date"
+      command:
+        cmd: /usr/bin/python -m pip install -U jinja2
+        creates: /usr/lib/python2.7/site-packages/jinja2/
```
Comment (sabre1041) [inline playbooks/prereqs.yml:15]: This would lock us into python2. Is that the desire?

**Review by sabre1041 — COMMENTED** [review]
One comment

File: playbooks/prereqs.yml (hunk 1 above)
Comment (rpelisse) [reply playbooks/prereqs.yml:15]: I want to say yes as Ansible uses python 2.7 by default on RHEL/CentOS system. Also, having both python3 and python2 is going to get us in trouble. I think it's better to be clean and say "only python2" (or only python3).

Comment (rpelisse) [reply playbooks/prereqs.yml:15]: Ah also, this is band aid. At some point, hopefully, the jinja2 package will get updated and a simply "yum instal python-jinja2" will do the trick, removing the need for this.

Comment (sabre1041) [reply playbooks/prereqs.yml:15]: merging as is to keep moving

**Review by sabre1041 — APPROVED** [review]
LGTM

**Pull request merged**